src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -178,6 +178,7 @@ export default function ActionsPanel(): JSX.Element {
     <Panel
       drawer={
         <ActionButton
+          className="actions-panel__action-button"
```

Comment:
You can also just use 

```suggestion
          className="u-no-margin--bottom"
```

Reply:
Thank you @fasih-mehmood, that's very useful to know about!